actions/release_component_test.go

```diff
@@ -0,0 +1,34 @@
+package actions
+
+import "testing"
```

Comment:
I've started writing those tests, but it requires resurrecting a bunch of mock code we had previously discarded, and building up several fake GH API objects. If we want to merge this in time for releasing Workflow v2.8.0, perhaps it would be better to add tests in a followup PR.